README.md

```diff
@@ -1,2 +1,80 @@
-# module-ballerina-math.vector
-Ballerina Math Module
+Ballerina Math Vector Library
```

Comment:
```suggestion
Ballerina `math vector` library
```
We use sentence case in the titles of all Ballerina documentation.

Reply:
Addressed in https://github.com/ballerina-platform/module-ballerina-math.vector/pull/1/commits/1fe0f828ef1ea29a4fc27924af5f541af69729ff.